Formula/mpv.rb

```diff
@@ -49,17 +49,21 @@ def install
       --datadir=#{pkgshare}
       --mandir=#{man}
       --docdir=#{doc}
+      --bashdir=#{bash_completion}
       --zshdir=#{zsh_completion}
       --lua=51deb
     ]
 
     system Formula["python@3.9"].opt_bin/"python3", "bootstrap.py"
     system Formula["python@3.9"].opt_bin/"python3", "waf", "configure", *args
     system Formula["python@3.9"].opt_bin/"python3", "waf", "install"
+    system Formula["python@3.9"].opt_bin/"python3", "TOOLS/osxbundle.py", "build/mpv"
+    prefix.install "build/mpv.app"
   end
 
   test do
     system bin/"mpv", "--ao=null", test_fixtures("test.wav")
     assert_match "vapoursynth", shell_output(bin/"mpv --vf=help")
+    system "#{opt_prefix}/mpv.app/Contents/MacOS/mpv", "--version"
```

Comment:
Any reason why this isn't being tested in the same way as `bin/"mpv"` above?

Reply:
I don't know what `vapoursynth` is, nor if it can be expected to always be in the help. Duplicate tests don't really add to the coverage. This is how tarsnap-gui.rb tests and that was a recent example of prefix.install an app that I could copy and paste from.

Reply:
I was actually referring to the first test line, not the second.

Version "tests" aren't particularly useful, since:
* they're often implemented as "short-circuits" in the app, and may not actually show issues with dynamically loading libraries or other app functionality that would be exercised with a proper run
* Homebrew tests usually don't show output except when tests fail, so you wouldn't normally see the version string anyway

Reply:
I don't know what tests are useful for mpv. This test just shows that there appears to be a MacOS app bundle. Even if short circuited it proves that there is a functional executable which is all I think is necessary. This follows the pattern of testing for several recently modified formulas that include MacOS bundles.

Reply:
> I don't know what tests are useful for mpv

For the bundle:
```
system "open", "-a" #{opt_prefix}/mpv.app", "--args", "--ao=null", test_fixtures("test.wav")
```
This helps verify that the app bundle will actually launch without error.

> Even if short circuited it proves that there is a functional executable which is all I think is necessary.

It only proves that there's a binary that can spit out its version. I just broke linkages to half a dozen of `mpv`'s dependencies and was still able to print the version...but not play anything.

Reply:
The first test confirms that mpv works, the third one just confirms that the app was created.